project/AppDependencies.scala

```diff
@@ -1,25 +1,20 @@
-import play.core.PlayVersion
 import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
   lazy val libraryDependencies = compile ++ test
 
-  val bootstrapVersion = "7.15.0"
+  val bootstrapVersion = "8.4.0"
 
   private lazy val compile = Seq(
     ws,
-    "uk.gov.hmrc"                   %% "bootstrap-backend-play-28"        % bootstrapVersion,
-    "com.typesafe.play"             %% "play-json"                        % "2.9.2",
-    "com.typesafe.play"             %% "play-json-joda"                   % "2.9.2",
-    "uk.gov.hmrc"                   %% "domain"                           % "8.1.0-play-28"
+    "uk.gov.hmrc"                   %% "bootstrap-backend-play-30"        % bootstrapVersion,
+    "org.playframework"             %% "play-json"                        % "3.0.2",
+    "uk.gov.hmrc"                   %% "domain-play-30"                   % "9.0.0"
   )
 
   private lazy val test = Seq(
-    "uk.gov.hmrc"                   %% "bootstrap-test-play-28"           % bootstrapVersion,
-    "org.mockito"                   %% "mockito-scala-scalatest"          % "1.17.29",
-    "org.scalatest"                 %% "scalatest"                        % "3.2.17",
-    "com.vladsch.flexmark"          %  "flexmark-all"                     % "0.62.2",
-    "com.github.tomakehurst"        %  "wiremock-jre8-standalone"         % "2.27.1"
+    "uk.gov.hmrc"                   %% "bootstrap-test-play-30"           % bootstrapVersion,
+    "org.mockito"                   %% "mockito-scala-scalatest"          % "1.17.29"
```

Comment:
```      "org.mockito"             %% "mockito-scala-scalatest"    % "1.17.14"      % Test,```
is provided by bootstrap-play

Reply:
If I remove it, there are many errors like:
```
[error] /.../api-platform-test/test-common/uk/gov/hmrc/util/HmrcSpec.scala:19:8: object ArgumentMatchersSugar is not a member of package org.mockito
[error] import org.mockito.{ArgumentMatchersSugar, MockitoSugar}

[error] /.../api-platform-test/test-common/uk/gov/hmrc/util/HmrcSpec.scala:27:105: not found: type MockitoSugar
[error] abstract class HmrcSpec extends AnyWordSpec with Matchers with OptionValues with WsScalaTestClient with MockitoSugar with ArgumentMatchersSugar

[error] /.../api-platform-test/test/unit/uk/gov/hmrc/apiplatformtest/controllers/AssetsControllerSpec.scala:40:38: not found: value mock
[error]     val mockAppContext: AppContext = mock[AppContext]

[error] /.../api-platform-test/test/unit/uk/gov/hmrc/apiplatformtest/controllers/AuthTestSupport.scala:32:5: not found: value when
[error]     when(mockAuthConnector.authorise[Unit](*, *)(*, *)).thenReturn(successful(()))

[error] /.../api-platform-test/test/unit/uk/gov/hmrc/apiplatformtest/controllers/AuthTestSupport.scala:32:44: not found: value *
[error]     when(mockAuthConnector.authorise[Unit](*, *)(*, *)).thenReturn(successful(()))
```